Libs/MRML/Core/vtkMRMLMarkupsNode.h

```diff
@@ -468,7 +468,7 @@ class  VTK_MRML_EXPORT vtkMRMLMarkupsNode : public vtkMRMLDisplayableNode
   void UnsetNthControlPointPosition(int pointIndex);
 
   /// Set control point status to ignored.
-  void SetNthControlPointPositionMissing(int pointIndex);
+  void SetControlPointPositionMissing(int pointIndex);
```

Comment:
```suggestion
/// Disable wrapping to prevent matching as a `SET_NTH` function.
VTK_WRAPEXCLUDE
void SetNthControlPointPositionMissing(int pointIndex);
```

Reply:
To ensure existing code works as expected, let's use the `VTK_WRAPEXCLUDE` hint.